dispatch/static/manager/src/js/api/dispatch.js

```diff
@@ -155,6 +155,14 @@ var DispatchAPI = {
     createTag: (token, name) => {
       return postRequest('tags', null, {name: name}, token)
     }
+  },
+  dashboard: {
```

Comment:
We're sort of breaking out of REST here, but I'm not totally opposed to that...

I'm just wondering if it then makes sense to combine both endpoints, or at least put them under the same root:

`GET api/dashboard/actions`
`GET api/dashboard/recent`

Thoughts?